spotify.md

```diff
@@ -0,0 +1,118 @@
+
+We rely on crowdsourcing and the feedback loop (as described above) to keep overall doc quality high. Our tech writers don’t review technical documentation.
+
+However, engineering teams do get word that we have tech writers stashed away on our team, and so we do get asked to help out with plenty of technical writing assignments. It is hard to say no sometimes, but overall we have managed to stick to our initial strategy:
+
+What we do offer is:
+
+* A one-hour introduction to technical writing at Spotify.
+* A one-hour “consultancy” meeting with the team.
+* A support channel for smaller questions.
+* Document templates.
+* A technical writing handbook that includes a style guide.
+
+We believe for a company the size of Spotify, this approach is the only way to scale quality technical documentation.
+
+## 10. Commit to docs like code and keep committing
+
+At Spotify, we have this key onboarding documentation called Golden Path tutorials. You can read about them here: [How we use Golden Paths to solve fragmentation in our software ecosystem](https://engineering.atspotify.com/2020/08/17/how-we-use-golden-paths-to-solve-fragmentation-in-our-software-ecosystem/). They present a great challenge to a docs-like-code solution because they are large, owned by multiple teams, and have many dependencies. In addition, we have one tutorial for each engineering discipline (for example, backend, data, web, etc.) and it’s crucial that the various tutorials play well together.
+
+These tutorials used to be in Google Docs and people used to very much appreciate this format in particular the strong commenting features. We could have easily opted to keep the Golden Path tutorials in Google Docs, but we had committed to docs like code and we wanted to go all in. I remember our product manager at the time saying something like: If we can make this work for the Golden Path tutorials, it will help us solve for all other technical documentation at Spotify.
+
+So we implemented the tutorials in TechDocs, use the [CODEOWNERS](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners) file to keep track of ownership, and set up a special GitHub bot to handle notifications when GitHub Issues are raised. We are also starting to take it even further by exploring ways that we can use tooling to help teams keep track of the documentation dependencies found in the tutorials.
+
+---
+
+So that’s about it. There’s your long-term relationship advice. Not bad for a bunch of techies eh?
+
+One last note to let you know that **we have now open sourced TechDocs**. Let’s take TechDocs to the next level together. Everything you need to know, you’ll find in the blog post [Announcing TechDocs: Spotify’s docs-like-code plugin for Backstage](https://backstage.io/blog/2020/09/08/announcing-tech-docs).
```

Comment:
This doesn't have to be the last note necessarily. :) I think it's a great lead in if you want it to be. I also think you could add a list of links here to all the technical resources so that others could see the entire stack your teams use.

Reply:
I get your point - the only place I could imagine it fitting in is as a tip or note box after the text "And most of the feedback we get from engineers is positive with the odd request for a small tweak or improvement." And before the diagram. And then it could say: By the way, we have open..."
Having said that, I also quite like it at the end. But we could change it to start like this...
"Oh yeh, almost forgot to say. We have now..."

Reply:
So all in all - if you feel my comments are satisfactory. Then I am happy for you to take it from here.

Reply:
Will do! I'll have a draft on the site ready for review next.